robottelo/cli/contentview.py

```diff
@@ -134,12 +134,12 @@ def version_info(cls, options, output_format=None):
         return result
 
     @classmethod
-    def version_incremental_update(cls, options):
+    def version_incremental_update(cls, options, format='csv'):
```

Comment:
I'd suggest to use output_format instead to keep this consistent with other CLI methods
```suggestion
    def version_incremental_update(cls, options, output_format='csv'):
```

Reply:
sure.